packages/arb-token-bridge-ui/src/hooks/useRedeemRetryable.ts

```diff
@@ -37,31 +37,42 @@ export function useRedeemRetryable(): UseRedeemRetryableResult {
       return
     }
 
-    let retryableTicket: IL1ToL2MessageWriter
-
     try {
       setIsRedeeming(true)
 
       if (!signer) {
         throw 'Signer is undefined'
       }
 
-      retryableTicket = await getRetryableTicket({
+      const retryableTicket = await getRetryableTicket({
         l1TxHash: tx.txId,
         retryableCreationId: tx.l1ToL2MsgData?.retryableCreationTxID,
         l1Provider: parentChainProvider,
         l2Signer: signer
       })
-    } catch (error: any) {
-      setIsRedeeming(false)
-      return errorToast(
-        `There was an error, here is more information: ${error.message}`
-      )
-    }
 
-    try {
-      const tx = await retryableTicket.redeem()
-      await tx.wait()
+      const reedemTx = await retryableTicket.redeem()
+      await reedemTx.wait()
+
+      const status = await retryableTicket.status()
+      const isSuccess = status === L1ToL2MessageStatus.REDEEMED
+
+      const redeemReceipt = (await retryableTicket.getSuccessfulRedeem()) as {
```

Comment:
Have to narrow down the type because of [this type in sdk](https://github.com/OffchainLabs/arbitrum-sdk/blob/main/src/lib/message/L1ToL2Message.ts#L236)